Sources/Components/SuggestShippingView/ShippingRequestErrorView.swift

```diff
@@ -0,0 +1,66 @@
+import Combine
+import FinniversKit
+
+public final class ShippingRequestErrorView: UIView {
+    private var viewModel: ShippingRequestErrorViewModel!
```

Comment:
This should be an optional, so we can avoid force unwraps.

If it can't be an optional, it should be provided through `init(viewModel:withAutoLayout:)`.